rtl/__init__.py

```diff
@@ -419,7 +419,10 @@ def copy_rtl_sources(self):
             # vlogsrc exists, simdut doesn't exist => copy vlogsrc to simdut
             elif vlogsrc_exists and not simdut_exists:
                 self.print_log(type='I', msg='Copying %s to %s' % (self.vlogsrc, self.simdut))
-                shutil.copyfile(self.vlogsrc, self.simdut, follow_symlinks=False)
+                if os.path.islink(self.vlogsrc):
+                    os.symlink(os.path.join(os.path.dirname(self.vlogsrc), os.readlink(self.vlogsrc)), self.simdut)
```

Comment:
Have you tested this with absolute symlinks? To me it looks like absolute symlinks would be malformed with this implementation.

Reply:
I did some testing. Looks like `os.path.join()` outputs its second argument if both paths are absolute so this should also work for absolute paths.